.gitignore

```diff
@@ -12,4 +12,7 @@
 *.out
```

Comment:
it would be nice to generate a new gitignore using gitignore.io

Reply:
Done